src/invite/InviteModal.tsx

```diff
@@ -42,7 +60,14 @@ const InviteModal = ({
       <View style={styles.contentContainer}>
         <Image style={styles.imageContainer} source={imageSource} resizeMode="contain" />
         <Text style={[fontStyles.h2, styles.text]}>{title}</Text>
-        <Text style={[fontStyles.regular, styles.text]}>{description}</Text>
+        {description ? <Text style={[fontStyles.regular, styles.text]}>{description}</Text> : null}
+        {descriptionKey ? (
+          <Text style={[fontStyles.regular, styles.text]}>
+            <Trans i18nKey={descriptionKey}>
```

Comment:
@jeanregisser are we still keeping the door open for automatically deriving translation keys, when we're not sure how it would work with Crowdin and OTA? if so, we should avoid separating the translation key from the translation function like this (but because we need the `Trans` component, we'd need to pass the whole component in as a prop :/ )